go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/tidepool-org/shoreline
 
-go 1.22
+go 1.23.0
+
+toolchain go1.24.1
```

Comment:
Is there some reason the toolchain version is different from the language version?